### PR TITLE
Fix freaks screen bugs

### PR DESCRIPTION
**Pull request by cipry13**
#### Description
Fix bugs on freaks screen

- [x]  Fix freak name not fully showing
- [x]  Fix freak card margins
- [x]  Names are now always centered in the freak card
- [x]  Refactor features.freaks packages

### CODE REVIEW

**Comment by cipry13** [comment]
[Fix freaks naming](https://trello.com/c/aqq9Pad0/210-fix-freaks-naming)

File: features/freaks/src/main/kotlin/com/agilefreaks/freaks_catalog/features/freaks/FreaksFragment.kt [hunk 1]
```diff
@@ -111,9 +111,8 @@ class FreaksFragment : Fragment() {
         mBottomSheetBinding.viewModel = filterViewModel
         mBottomSheetBinding.filterTitle.text = name
 
-        val recyclerFiltersView =
-            mBottomSheetBinding.recyclerFiltersView
-        recyclerFiltersView.layoutManager = LinearLayoutManager(requireContext())
+        mBottomSheetBinding.recyclerFiltersView.layoutManager =
```
Comment (AlexandraDamaschin) [inline features/freaks/src/main/kotlin/com/agilefreaks/freaks_catalog/features/freaks/FreaksFragment.kt]: Why are we naming variables with m_____? 
I like a lot this comment about this subject: https://stackoverflow.com/a/26563953/9648176
What are your thoughts about this?

**Review by AlexandraDamaschin — CHANGES_REQUESTED** [review]
One small improvement

**Review by AlexandraDamaschin — APPROVED** [review]
Does this improve software quality? Yes

**Pull request merged**